### DISCUSSION TITLE
Verinet getting stuck in DNNV

### DISCUSSION THREAD

**Discussion by AWbosman** [dlshriver/DNNV#106] [Q&A] [answered]
Hi everyone!

Currently I am using VeriNet within the DNNV framework to verify a bunch of different properties. I've noticed that on some properties VeriNet seems to get stuck in part of the search space. 
Is there anyone using DNNV that found a solution to this?
I've been thinking about maybe parallellising the search tree, but I am not sure whether this is possible on cpu and within DNNV. 

Thanks in advance

**Comment by dlshriver** [comment, marked as answer]
Unfortunately, the version of VeriNet supported by DNNV is quite old at this point and doesn't always seem to work for some reason. The error in #101 may be related. I have not been able to diagnose the cause, but I don't think I'm going to put much effort into trying to fix such an old version of the verifier. Hopefully someday we can add support for the newest version of VeriNet, which should be much better, but I don't have the cycles for that at the moment. However, I would be happy to accept a pull request from anyone who wants to add it.
